### ISSUE TITLE
Feature request: Add ringtone alarm to the group channels

### ISSUE THREAD

**Issue by kaksikanaa** [mattermost/mattermost-plugin-calls#621]
Hello!

We are aware of the considerations about the subject (for example here https://github.com/mattermost/mattermost-plugin-calls/issues/125) but still in our use cases we do need the ringtone to the channel group calls also (like in DM channels).

It would be wonderful if this could be a option for the admin to set and even more awesome if then the option could be still configurable per channel bases.

 
![Admin-setting](https://github.com/mattermost/mattermost-plugin-calls/assets/15322015/47fd6f38-eeea-4400-b5e4-5e1a596815e6)
![Channel-setting](https://github.com/mattermost/mattermost-plugin-calls/assets/15322015/76b95187-01ac-416b-9ed7-43c4f21b0e4d)

**Comment by Judx** [comment]
Up! I've often found myself in situations where I have missed a GM call because it does not ring. The proposed feature, channel configurable ring tone, would be a significant improvement for users like myself.

**Comment by streamer45** [comment]
Thanks for the feedback @kaksikanaa!

@cpoile Shouldn't ringing apply to GMs as well?

**Comment by cpoile** [comment]
@streamer45 It does -- does ringing happen for DMs and not for GMs @Judx ? If so, can you check to make sure you haven't muted the GM channel or turned off desktop notifications in that channel?

**Comment by Judx** [comment]
So sorry I think I understood the naming wrong. I am part of over 10 private channels I use every day. When someone calls to those channels there is no ringing. However "channels" under DM ring. Is this right?

DM = Direct Message -> Rings
GM = Group Message -> Rings
CM = Channel Message -> Does not ring

**Comment by streamer45** [comment]
Yep that's correct. I think the plan was to eventually extend the functionality to all channels (public and private) but this likely need some extra UX functionality (e.g. calls invitees) since we don't want to be ringing for a whole channel in which only a subset of members may be interested in joining a given call.

/cc @matthewbirtch

**Comment by matthewbirtch** [comment]
> Yep that's correct. I think the plan was to eventually extend the functionality to all channels (public and private) but this likely need some extra UX functionality (e.g. calls invitees) since we don't want to be ringing for a whole channel in which only a subset of members may be interested in joining a given call.
> 
> /cc @matthewbirtch

Agree. This would be very problematic for channels with large membership, so if we were to do this, we would need a way to limit the ringing only to those interested. As @streamer45 mentioned, having the ability to have invitees to a call would likely be the only way to do this. Scheduling and inviting people to calls is something on the radar, but not as an immediate priority right now.

**Comment by Judx** [comment]
Indeed large channels become problematic if ringing for channels is enabled and some channels are not meant for calling at all. At least in theory we could overcome spam by:
- New channel setting: "Ring for calls" , default off.
- New user setting: "Ring for channels", default off
- New user based channel setting: "Ring for channel", default off

**Comment by ageekhere** [comment]
I also up this feature, also would be cool to have the ringtone on the mobile apps as well.

**Comment by bdodds** [comment]
Hello, chiming in as a paid Mattermost customer, I'd love to see more options to ringing channels.  We're ultimately looking to replace Skype and Signal and things like that, and they currently have a bit more flexibility in the ringing everyone department - at least in Skype.  In Skype, I can start a call in a channel with 30 people in it, but not ring everyone.  But, if I want to, I could hit the ring button and it'll ring everyone in the channel.  We use that for scrums and standups every morning in large groups. Personally I appreciate the elegance of the calls plugin the way it is, but other folks at my company would like to have the option to make ringing way more persistent and universal.  Thanks for your consideration, I appreciate all of the work that goes into making this a robust and useful solution.

**Comment by Judx** [comment]
I have opened pull request to add this feature.
